boards/arm/mimxrt685_evk/mimxrt685_evk_cm33.dts

```diff
@@ -60,6 +60,18 @@
 			label = "User LED_RED";
 		};
 	};
+
+	power-states {
+		idle: idle {
+			compatible = "zephyr,power-state";
+			power-state-name = "runtime-idle";
+			min-residency-us = <1000>;
```

Comment:
Would be good in the commit message to explain how/where the 1000 comes from.

Reply:
I am looking for some guidance on this. I picked a value based on what I saw was done for other platforms. I have asked the slack power-management channel as well.

Reply:
Thank you. I have update the `min-residency-us` value and added information in the commit message.